plugins/org.jboss.tools.ws.jaxrs.ui/plugin.xml

```diff
@@ -636,5 +636,16 @@
       </perspectiveExtension>
    </extension>
 
-       
+   <extension
+         point="org.eclipse.ui.ide.projectConfigurators">
+      <projectConfigurator
+            class="org.jboss.tools.ws.jaxrs.ui.internal.importer.JaxRsConfigurator"
+            label="Java EE Jax-RS">
+         <activeWhen>
+            <hasFileRecursively
+                  filename="web.xml">

```

Comment:
The `web.xml` deployment descriptor is not mandatory in Java EE 6+ projects. Does it have any impact here ?